cmd/amp/account.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Interactive
+	loginCmd = &cobra.Command{
+		Use:   "login",
+		Short: "log in to amp",
+	}
+
+	accountCmd = &cobra.Command{
+		Use:   "account",
+		Short: "account operations",
+	}
+	// The rest of the commands are subcommands
+
+	// Interactive
+	signUpCmd = &cobra.Command{
+		Use:   "signup",
+		Short: "Create a new account and login",
+	}
+
+	// Interactive
+	verifyCmd = &cobra.Command{
+		Use:   "verify [CODE]",
+		Short: "verify email using code",
+	}
+
+	assumeRoleCmd = &cobra.Command{
+		Use:   "assume [ORGANIZATION] [TEAM]",
+		Short: "assume role as organation or team",
+	}
+
+	createOrganizationCmd = &cobra.Command{
+		Use:   "create organization [NAME] [EMAIL]",
+		Short: "Create an organization",
+	}
+
+	listUsersCmd = &cobra.Command{
+		Use:   "list users [ORGANIZATION] [TEAM]",
+		Short: "list users, optionally filter by organation and team",
+	}
+
+	listOrganizationsCmd = &cobra.Command{
+		Use:   "list organizations",
+		Short: "list organizations",
+	}
+
+	listTeamsCmd = &cobra.Command{
+		Use:   "list teams [ORGANIZATION]",
+		Short: "list teams by organization",
+	}
+
+	listPermissionsCmd = &cobra.Command{
+		Use:   "list permissions [ORGANIZATION] [TEAM]",
+		Short: "list permissions by team",
+	}
+
+	infoCmd = &cobra.Command{
+		Use:   "info [name]",
+		Short: "list account information",
+	}
+
+	editCmd = &cobra.Command{
+		Use:   "edit [name]",
+		Short: "edits account information",
+	}
+
+	deleteCmd = &cobra.Command{
+		Use:   "delete [name]",
+		Short: "deletes account",
+	}
+
+	addOrganizationMembersCmd = &cobra.Command{
+		Use:   "add organation members [ORGANIZATION] [MEMBERS...]",
```

Comment:
We don't have a notification/accept system defined so far.  We should create some user stories around it.